content/docs/06.enterprise/02.governance/01.assets.md

```diff
@@ -5,305 +5,239 @@ editions: ["EE", "Cloud"]
 version: ">= 1.2.0"
 ---
 
-Define, track, and manage metadata about resources created or used by Kestra flows.
+Track and manage the resources your workflows create and use.
+
 
 ## Overview
 
-Assets can be created automatically by tasks or manually from the **Assets** UI page. From this page, users can view and edit asset metadata, inspect **Executions** that created or updated the asset, and explore a **Dependencies** graph showing relationships between assets, flows, and apps.
+Assets keeps a live inventory of resources that your workflows interact with. These resources can be database tables, virtual machines, files, or any external system you work with.
+
+You create assets in two ways: automatically through task execution or manually from the Assets UI. Once created, you can view asset details, check which workflow runs created or modified them, and see how assets connect to each other across your workflows.
 
 This feature enables:
 
 - Shipping metadata to lineage providers (e.g., OpenLineage).
 - Populating dropdowns or Pebble inputs with live assets (e.g., available VMs).
-- Monitoring assets and their state from custom dashboards.
-- Reacting to asset changes (e.g., VM powered on/off) that emit events.
+- Monitoring assets and their state.
 
 ## Asset definition
 
-Assets can be defined directly on a task using the `assets` array property or created dynamically using a `Materialize` task.
+Define assets directly on any task using the `assets` property. Each task can declare input assets (resources it reads) and output assets (resources it creates or modifies).
+
+Every asset includes these fields:
 
 | Field | Description |
 | --- | --- |
-| `id` | unique within a namespace |
-| `namespace` | each asset is tied to a namespace for RBAC reasons |
-| `type` | free-form string used for filtering, must be indexed |
+| `id` | unique within a tenant |
+| `namespace` | each asset can be associated with a namespace for filtering and RBAC management |
+| `type` | free-form string used for filtering |
```

Comment:
It is described lower, but we should make it clear, that this can either be one of the predefined types, or a free form type. I was surprised to see `io.kestra.core.models.assets.Table` in the example right below, as it wasn't explained, that there are predefined types.